Libplanet/Action/ActionContext.cs

```diff
@@ -30,5 +32,15 @@ internal class ActionContext : IActionContext
         public IAccountStateDelta PreviousStates { get; }
 
         public IRandom Random { get; }
+
+        public Guid Guid
+        {
+            get
+            {
+                var b = new byte[16];
```

Comment:
I think that we should create GUID based [rfc4122](https://www.ietf.org/rfc/rfc4122.txt) like [dotnet-core](https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/System/Guid.Unix.cs#L12-L35).

Reply:
I think implementing rfc4122 takes a lot of time, so it's hard to apply it in this PR.